packages/framework/src/evaluate/binding.ts

```diff
@@ -88,6 +89,7 @@ export const createBindingAtom = <T = unknown>(
         },
         env: get(envAtom),
         secrets: get(secretAtom),
+        useMockResponse: useMockResponse
```

Comment:
This is likely unneeded since this is primary for read expressions but for completeness you will need to create a read-only atom with the value and call get on the atom here so the binding automatically updates.